### PR TITLE
Update defaults to 61

### PR DESCRIPTION
**Pull request by stevieing**
Closes #

#### Changes proposed in this pull request

- Update rails defaults to 6.1

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -9,7 +9,8 @@
 
 module Labwhere
   class Application < Rails::Application
-    config.load_defaults 5.2
+    config.load_defaults 6.1
+    config.autoloader = :zeitwerk
```
Comment (BenTopping) [inline config/application.rb:13]: I think zeitwerk is the default so might be able to remove line 13.

File: config/application.rb
Comment (BenTopping) [inline config/application.rb:1]: Can we remove the autoload_paths references now we are using zeitwerk?

File: config/application.rb [hunk 2]
```diff
@@ -45,6 +46,11 @@ class Application < Rails::Application
 
     config.label_templates = Rails.application.config_for(:label_templates)
 
+    config.eager_load_paths += %W[#{config.root}/app/lib/utils #{config.root}/app/lib/validators
```
Comment (BenTopping) [inline config/application.rb:49]: Are these required because their namespacing isn't right and zeitwerk can't find it properly?

**Review by BenTopping — CHANGES_REQUESTED** [review]

File: config/application.rb
Comment (stevieing) [reply config/application.rb:1]: Not as far as I am aware. We still require them in traction. I will probably look at this again later I just needed to stop somehwere.

File: config/application.rb (hunk 2 above)
Comment (stevieing) [reply config/application.rb:48]: Yes. Labwhere has some strange folder structure. As per above I wanted to stop somewhere. I will probably look at these again when I update some of the other dependencies.

**Review by BenTopping — APPROVED** [review]

**Pull request merged**